arm/Microsoft.Automanage/accounts/deploy.bicep

```diff
@@ -33,51 +18,49 @@ module pidName '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
   params: {}
 }
 
-module autoManageAccount '.bicep/nested_autoManageAccount.bicep' = {
-  name: '${uniqueString(deployment().name, location)}-AutoManageAccount'
-  scope: resourceGroup(autoManageAccountResourceGroupName)
-  params: {
-    location: location
-    autoManageAccountName: name
+resource account 'Microsoft.Automanage/accounts@2020-06-30-preview' = {
```

Comment:
tbd offline if the automanage account is still needed
Ref: https://docs.microsoft.com/en-us/azure/automanage/automanage-upgrade#automanage-account-and-first-party-application

Reply:
Updated to latest -> Deployment inside VM module